DependencyInjection/SonataMediaExtension.php

```diff
@@ -548,6 +553,38 @@ public function configureClassesToCompile()
     }
 
     /**
+     * Sets category and category manager in media bundle.
+     *
+     * @param ContainerBuilder $container
+     * @param array            $config
+     */
+    private function configureCategoryInMedia(ContainerBuilder $container, array $config)
+    {
+        if (!$this->isClassificationEnabled($config)) {
+            return;
+        }
+
+        $container->setAlias('sonata.media.manager.category', $config['category_manager'] ?: 'sonata.media.manager.category.default');
+
+        if (null === $config['class']['category']) {
+            $config['class']['category'] = 'Application\\Sonata\\ClassificationBundle\\Entity\\Category';
+        }
+    }
+
+    /**
+     * Checks if the classification of media is enabled.
+     *
+     * @param array $config
+     *
+     * @return bool
+     */
+    private function isClassificationEnabled(array $config)
+    {
+        return !$config['force_disable_category'] &&
+            (null !== $config['category_manager'] || interface_exists('Sonata\ClassificationBundle\Model\CategoryInterface'));
```

Comment:
Are you sure? on master is with an `||` and it makes sense because this line makes a default if category_manager is not defined:

```
$container->setAlias('sonata.media.manager.category', $config['category_manager'] ?: 'sonata.media.manager.category.default');
```